mftf/2.3/tips-tricks.md

```diff
@@ -8,6 +8,44 @@ Sometimes, little changes can make a big difference in your project. Here are so
 
 ## Actions and action groups
 
+## Use parameterized selectors in action groups with argument references
```

Comment:
Is this meant to be a section under **Actions and action groups**? 

If so it should be `###`.
```suggestion
### Use parameterized selectors in action groups with argument references
```